GliaWidgets/Sources/Layout/Layoutable.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+/// Defines layout specific properties. This interface helps to make
+///  flexible mechanism for defining UI and avoid make strict relations
+///  with UIView.
+///
```

Comment:
Here the comments spacing is not even, and also there's a space before and after the protocol declaration.

Reply:
fixed  - 5966c5650a8fdf420523c179f24a8d4d0c741918